### DISCUSSION TITLE
About random seed

### DISCUSSION THREAD

**Discussion by kmc7468** [ANLAB-KAIST/KENSv3#226] [Project Q&A 2. Transport Layer (TCP)] [answered]
I found that my and the solution implementation fail when random seed is one of the followings: 4054843009, 2279280275, 912872428 (and probably there are more such random seeds)

Can I assume that the random seed where the solution fails will not be used to grading?

**Comment by Pusnow** [comment, marked as answer]
We will exclude random seeds that solution fails to handle.
